src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHelpers.cs

```diff
@@ -113,11 +114,55 @@ internal static bool IsAllNullBytes(Span<byte> buffer)
             return true;
         }
 
-        // Returns a DateTimeOffset instance representing the number of seconds that have passed since the Unix Epoch.
-        internal static DateTimeOffset GetDateTimeFromSecondsSinceEpoch(double secondsSinceUnixEpoch)
+        // Converts the specified number of seconds that have passed since the Unix Epoch to a DateTimeOffset.
+        internal static DateTimeOffset GetDateTimeOffsetFromSecondsSinceEpoch(double secondsSinceUnixEpoch) =>
```

Comment:
A lot of the callers to this method pass `int` and `long`. Would it make sense to just have an overload take a `long` and not have to do the conversion back and forth?

Reply:
Ah, true.

The TarWriter.Unix.cs calls take a long. <- This takes a timestamp (mtime, atime or ctime) that comes from a FileStatus struct that came from LStat.
The TarHeader.Read.cs calls take an int. <- This takes a timestamp that comes from a GnuTarEntry (AccessTime or ChangeTime).
The TarHelpers.cs call takes a double. <- This takes a timestamp that comes from a PAX extended attributes KVP, which is originally a string representing a double number.